Changelog 1.10.0 (#5004)
btcpayserver/btcpayserver: Changelog.md

# Changelog

## 1.10.0

### New features

* In addition to the `Owner` and `Guest` role available for users of a store, it is now possible to create new custom roles and to adjust the permissions granted by `Owner` and `Guest`. (#4940) @Kukks
* Forms: It was only possible to configure a forms via some JSON configuration, we now have a nice UI editor for it (#4968) @dennisreimann @dstrukt

> * Hide a toggle to mask sensitive info, "the lurking wife mode" (#4966) @dennisreimann

Reviewer: ```suggestion
* Setting to hide sensitive info, such as balances and amounts (#4966) @dennisreimann
```

Author: party pooper

* Forms: Add multiline input (#4942) @dennisreimann
* In the refund workflow, make it easier to only reimburse overpaid amounts (#4934 #4812) @dennisreimann

### Bug fixes

> * Fix: If using LNBank, payments would sometimes not be detected by BTCPay Server. (https://github.com/dennisreimann/btcpayserver-plugin-lnbank/issues/33) @NicolasDorier

Reviewer: ```suggestion
* Fix: When using LNbank, payments would sometimes not be detected by BTCPay Server (dennisreimann/btcpayserver-plugin-lnbank#33) @NicolasDorier
```
